packages/react-components/lib/doors/door-table.tsx

```diff
@@ -10,20 +10,48 @@ import {
 } from '@material-ui/core';
 import React from 'react';
 import * as RmfModels from 'rmf-models';
+import { FixedSizeList, ListChildComponentProps } from 'react-window';
 import { DoorData, doorModeToString, doorTypeToString } from './utils';
+import clsx from 'clsx';
+import AutoSizer from 'react-virtualized-auto-sizer';
 
 export interface DoorTableProps {
   doors: DoorData[];
   doorStates: Record<string, RmfModels.DoorState>;
   onDoorControlClick?(event: React.MouseEvent, door: RmfModels.Door, mode: number): void;
 }
 
+export interface DoorFixListDataProps extends DoorTableProps {
```

Comment:
```suggestion
interface DoorFixedListData extends DoorTableProps {
```

No export because this shouldn't be used by other components.
No props suffix because this is not used as a prop directly.